src/board.cpp

```diff
@@ -237,4 +243,36 @@ float getBatteryVoltage()
   Serial.println("Battery voltage: " + String(volt) + " V");
   return volt;
 }
+
+unsigned long checkButtonPressDuration()
+{
+#ifdef EXT_BUTTON
```

Comment:
Should be here a case when pull-up is not used? If yes I wouldn't use this macro.